main/src/cgeo/geocaching/network/Network.java

```diff
@@ -223,6 +223,15 @@ private Network() {
         return RxOkHttpUtils.request(OK_HTTP_CLIENT, request.build());
     }
 
+    @NonNull
+    public static <T> Single<T> postJsonRequest(final String uri, final Class<T> clazz, final Object jsonObject) throws JsonProcessingException {
+        final Builder request = new Builder().url(uri).post(RequestBody.create(MEDIA_TYPE_APPLICATION_JSON,
+                mapper.writeValueAsString(jsonObject)));
+        Single<Response> response = RxOkHttpUtils.request(OK_HTTP_CLIENT, request.build());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'response' should be declared final.](https://app.codacy.com/manual/cgeo/cgeo/pullRequest?prid=5262679)